.vscode/launch.json

```diff
@@ -0,0 +1,49 @@
+{
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"name": "Launch (TD)",
+			"type": "clr",
+			"linux": {
```

Comment:
I think we have to add "osx" specific overrides too
```
"osx": {
	"type": "mono"
},
```

Reply:
Yes, confirmed that this is needed and works (to a limited degree: all mods NRE in `DropDownButtonWidget` when I try to run them on this branch)

Reply:
Okay, untested though.

Reply:
Ok, it looks like the debug task just doesn't force a compile first... weird choice there vscode.